src/components/Events/Fall_GM/Fall_GM.js

```diff
@@ -5,57 +5,196 @@ import Countdown from 'react-countdown';
-			let secondString = "Second";
-	
-			if (days !== 1) dayString += 's';
-			if (hours !== 1) hourString += 's';
-			if (minutes !== 1) minuteString += 's';
-			if (seconds !== 1) secondString += 's';
+		const FALL_GM_2020_START_TIME = new Date('2020-10-05T18:30:00-0700'); //October 5th 2020 at 6:30 PDT
+
+		const renderCountdownOrNow = ({ days, hours, minutes, seconds, completed }) => {
+			if (completed) return <GM_NOW/>;
+			else {
+				let dayString = "Day";
+				let hourString = "Hour";
+				let minuteString = "Minute";
+				let secondString = "Second";
+				if(days !== 1) dayString += "s";
+				if(hours !== 1) hourString += "s";
+				if(minutes !== 1) minuteString += "s";
+				if(seconds !== 1) secondString += "s";
+
+				return <GM_COUNTDOWN days={days} hours={hours} minutes={minutes} seconds={seconds}
+				dayString={dayString} hourString={hourString} minuteString={minuteString} secondString={secondString}/>
+			}
+		};
 	
+		return (
+			<div id="fall-gm-page">
+				<Banner decorative />
+				<Countdown date={FALL_GM_2020_START_TIME} renderer={renderCountdownOrNow} />
+			</div>
+		);
+	}
+}
+
+class GM_NOW extends React.Component{
+	renderCommitteeSpeakers() {
+		const committeeData = [
+			{
+				id: "studio",
+				displayName: "ACM Studio",
+				speakers: "Kade Adams, Evan Charfauros",
+				logo: "/images/committees/logo-studio.svg",
+			},
+			{
+				id: "icpc",
+				displayName: "ACM ICPC",
+				speakers: "Nikil Selvam",
+				logo: "/images/committees/logo-icpc.svg",
+			},
+			{
+				id: "design",
+				displayName: "ACM Design",
+				speakers: "Haki Alpay, Tomoki Fukazawa",
+				logo: "/images/committees/logo-design.svg",
+			},
+			{
+				id: "cyber",
+				displayName: "ACM Cyber",
+				speakers: "Disha Zambani, Ellie Cheng",
+				logo: "/images/committees/logo-cyber.svg",
+			},
+			{
+				id: "teach",
+				displayName: "ACM TeachLA",
+				speakers: "Matt Wang",
+				logo: "/images/committees/logo-teachla.svg",
+			},
+			{
+				id: "w",
+				displayName: "ACM W",
+				speakers: "Julia Baylon",
+				logo: "/images/committees/logo-w.svg",
+			},
+			{
+				id: "ai",
+				displayName: "ACM AI",
+				speakers: "John Dang",
+				logo: "/images/committees/logo-ai.svg",
+			},
+			{
+				id: "hack",
+				displayName: "ACM Hack",
+				speakers: "Raji Jadhav",
+				logo: "/images/committees/logo-hack.svg",
+			},
+		]
+		return committeeData.map((committee) => {
 			return (
-				<div id="time-to-gm-wrapper">
-					<h1 id="time-to-gm-headline">Fall General Meeting 2020 begins in</h1>
-					<div id="countdown-wrapper">
-						<div className="countdown-cards">
-							<div className="square-background"/>
-							<div className="countdown-numbers">{days}</div>
-							<div className="countdown-labels">{dayString}</div>
+			<p><span id={committee.id}><img src={process.env.PUBLIC_URL + committee.logo} alt={"Logo for " + committee.displayName} className="committee-logos"/>{committee.displayName}</span>: {committee.speakers}</p>
```

Comment:
I'd put this massive statement onto multiple lines for better readability. Also, use object destructuring on the `committee` object
```suggestion
const { id, logo, displayName, speakers } = committee;
			<p><span id={id}><img src={`${process.env.PUBLIC_URL}{logo}} alt={`Logo for ${displayName}`} className="committee-logos" />{displayName}</span>: {speakers}</p>
```

Reply:
looks much cleaner now, thanks!